cmd/device-hub/cmd_server.go

```diff
@@ -29,6 +30,12 @@ var serverCommand = &cobra.Command{
 
 		var dataImpl store.Storer
 
+		// Load Config File if is enabled
+		if _config.ConfigFile {
+			log.Println("Overriding all settings with config file")
+			_config.AddConfigFile(_config.ConfigPath)
+		}
+
```

Comment:
Once it is moved could the log line list which keys are being overridden? 
From my testing omitted keys from the config file are not overridden so I don't think the message is correct.